kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/api/Configuration.java

```diff
@@ -54,6 +54,12 @@
     String LOGS_COPY = "logs.copy";
     String LOGS_PATH = "logs.path";
 
+    String USERNAME = "name";
```

Comment:
`username` to match?

Reply:
We are looking for system property and env variable. If we used `username`, then `System.getenv("USERNAME")` returns your host name. 

I tried `user.name` as well, but `System.getProperty("user.name")` returns your host name. thats why I am using `name`

Reply:
How about we prefix everything with `arq.cube` to have our own namespace? @lordofthejars wdyt?

Reply:
I don't know. I am neither so sure about it, maybe leave it as Diapk is the best approach now.

Reply:
I will keep it the same for now.